wsmaster/che-core-api-devfile/src/main/resources/schema/devfile.json

```diff
@@ -30,8 +30,15 @@
   },
   "required": [
     "specVersion",
-    "name",
-    "components"
+    "name"
+  ],
+  "anyOf": [
```

Comment:
Not sure this is needed. Given the default-editor logic, an empty devfile (i.e. one with no projects and no components) would just be provisioned with the default editor and its supporting plugins. Which I think is a workable environment, isn't it?

Reply:
It's kind of strange case to create a workspace with empty Devfile
```
---
specVersion: 0.0.1
name: my-empty-ws
```
But I agree that it makes sense. User will be able to use a workspace with default editor and import projects there. 
Will remove this `anyOf`